### ISSUE TITLE
Using tf1.4 to restore a model from tf0.8, a NotFoundError appeared

### ISSUE THREAD

**Issue by JingzheLyp** [tensorflow/tensorflow#16110]
When using tf1.4 to restore a model from tf0.8, I met a NotFoundError, the related code as flow:

ema = tf.train.ExponentialMovingAverage(1.0)
saver = tf.train.Saver(ema.variables_to_restore())
model_checkpoint_path='./model_check_point/model-20160506.ckpt-500000'
saver.restore(sess, model_checkpoint_path)

The error as flow:
NotFoundError (see above for traceback): Tensor name "incept3a/in3_conv5x5_8/batch_norm/moments/Squeeze/ExponentialMovingAverage" not found in checkpoint files ./model_check_point/model-20160506.ckpt-500000
	 [[Node: save/RestoreV2_46 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_46/tensor_names, save/RestoreV2_46/shape_and_slices)]]
	 [[Node: save/RestoreV2_315/_35 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_702_save/RestoreV2_315", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

How can I solve this problem?

This is a project I download from the github, the code is not writen by me. I run it on ubuntu14.04,
with TF1.4.1 installed from source code, bazel0.8 , CUDA8.0, cudnn6.0, GTX1060 6G memory.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by michaelisard** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
